### ISSUE TITLE
oxfmt: `sortImports` should sort named specifiers within import statements

### ISSUE THREAD

**Issue by kennyheard** [oxc-project/oxc#20160]
`sortImports` sorts the order of import statements but doesn't sort named specifiers within the braces.

```typescript
// Input
import { useState, useEffect, useRef } from "react";

// Expected after formatting
import { useEffect, useRef, useState } from "react";

// Actual after formatting — unchanged
import { useState, useEffect, useRef } from "react";
```

Biome handles this as part of its `organizeImports` feature in a single pass.

Currently, the only way to get specifier sorting in the Oxc ecosystem is to enable the `eslint/sort-imports` linter rule in oxlint alongside oxfmt's `sortImports`, with `ignoreDeclarationSort: true` to avoid the two tools conflicting over statement order. This works but requires coordinating two tools for what should be a single formatter concern.

This gap has downstream effects — [Ultracite](https://github.com/haydenbleasel/ultracite) (a popular preset provider) [disabled the `sort-imports` linter rule entirely](https://github.com/haydenbleasel/ultracite/pull/507) to avoid conflicts with oxfmt, leaving users with no specifier sorting at all.

**Comment by leaysgur** [comment]
Thanks for the feedback.

We're aware of the demand for sorting in import-named and are currently gathering interest here.
